ci-operator/step-registry/ipi/gcp/pre/custom-dns/ipi-gcp-pre-custom-dns-chain.metadata.json

```diff
@@ -0,0 +1,12 @@
+{
+	"path": "ipi/gcp/pre/custom-dns/ipi-gcp-pre-custom-dns-chain.yaml",
+	"owners": {
+		"approvers": [
+			"jianlinliu",
+			"patrickdillon",
+			"technical-release-team-approvers",
+			"vrutkovs",
+			"yunjiang29"
```

Comment:
Lets limit this and the owners files just to the installer team, so drop vadim, trt, david and anyone else not from our direct team